packages/itwin/property-grid/src/components/PropertyGrid.tsx

```diff
@@ -160,6 +162,21 @@ export const PropertyGrid = ({
     };
   }, []);
 
+  // If persisting hide/show empty values, get the preference
+  useMemo(() => {
+    const getPreferences = async () => {
+      if(persistNullValueToggle) {
+        const res = await getShowNullValuesPreference();
+        if(res !== undefined) {
```

Comment:
format changes please

Reply:
I'm not sure what you mean here. Everything passes the linter.

Reply:
look at the `if(` it should be `if (` and this should happen automatically when you save the file if you have the settings found in `.vscode/settings.json` here running:
```json
"editor.formatOnSave": true,
  ```
  perhaps something is overriding that for you